### DISCUSSION TITLE
OpenCV homebre install create extra folder which is not present in the original build

### DISCUSSION THREAD

**Discussion by ksajan** [Homebrew/discussions#3034] [Writing Formulae/Casks]
I was trying to install OpenCV for running it in golang. But it appears to me that when I install OpenCV through brew it creates an extra folder called`opencv4` inside the `include` folder. I have checked with the original source code builds it's not the case there if I build from source or directly download the release. I have installed the latest 4.5.5 version of OpenCV. 
This makes many of the GOCV libraries in go to not run at all. Can you please let me know If this is only for my case or others can also see this happening? I am using MacBook with macOS Monterey 12.2.1.

```
$brew config
HOMEBREW_VERSION: 3.4.0
ORIGIN: https://github.com/Homebrew/brew
HEAD: ffc76654da12fba7d34d078fd9bf681fef65921e
Last commit: 10 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 59cf0068039b16fab9d24081ef8ac30f5a110a4a
Core tap last commit: 31 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit kabylake
Clang: 12.0.5 build 1205
Git: 2.32.0 => /usr/local/bin/git
Curl: 7.77.0 => /usr/bin/curl
macOS: 12.2.1-x86_64
CLT: 12.5.0.22.9
Xcode: 13.2.1```

**Comment by cho-m** [comment]
I think that is expected as there is some upstream documentation that mentions the structure like https://github.com/opencv/opencv/blob/4.x/doc/tutorials/introduction/linux_install/linux_install.markdown#install
> By default OpenCV will be installed to the /usr/local directory, all files will be copied to following locations:
> ...
> - `/usr/local/include/opencv4` - headers

And it is how most Linux distros also package it like:
- Debian https://packages.debian.org/sid/amd64/libopencv-contrib-dev/filelist
- ArchLinux https://archlinux.org/packages/extra/x86_64/opencv/

The corresponding CMake and pkg-config files should include this information for builds that use these tools.

I see that GoCV mentions it uses `pkg-config`, so can you check the output of that for opencv?
